lib/pbench/server/__init__.py

```diff
@@ -157,29 +158,63 @@ def _ref_datetime(self):
 
     @property
     def rest_uri(self):
-        return self._get_conf("pbench-server", "rest_uri")
+        return self.get_conf("pbench-server", "rest_uri")
```

Comment:
How that would work since a option is always in a section?  Could you propose a PR for what you are thinking?  I really don't want to expand this PR beyond what it is.

Reply:
See [my comment to #3172](https://github.com/distributed-system-analysis/pbench/pull/3172#discussion_r1071310059).